src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -7,57 +7,82 @@
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.skin.Skin;
+import com.wikia.webdriver.common.skin.SkinHelper;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.elements.mercury.components.Search;
 import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
 import com.wikia.webdriver.elements.mercury.pages.SearchResultsPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.DiscussionsPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
 
 import org.testng.annotations.Test;
 
 @Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
-@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+@InBrowser(browser = Browser.CHROME)
 @Test(groups = "MercurySearch")
 public class SearchTests extends NewTestTemplate {
 
   private static final String SEARCH_PHRASE = "Infobox";
   private static final String SEARCH_PHRASE_NO_RESULTS = "AComplexQueryWithNoResults";
 
-  @Test(groups = "mercury_search_navigateToPageUsingSearchSuggestions")
-  public void mercury_search_navigateToPageUsingSearchSuggestions() {
-    String suggestionLink =
-        new ArticlePage()
-            .open(MercurySubpages.MAIN_PAGE)
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnMobile")
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
+    ArticlePage article = new ArticlePage().open(MercurySubpages.MAIN_PAGE);
+    String clickedSuggestion = article.getTopBar()
+        .openSearch()
+        .typeInSearch(SEARCH_PHRASE)
+        .clickSearchSuggestion(0);
+
+    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MERCURY));
+    Assertion.assertEquals(clickedSuggestion.toLowerCase(),
+                           article.getHeader().getPageTitle().toLowerCase());
+  }
+
+  @Execute(onWikia = MercuryWikis.MEDIAWIKI_119)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnDesktop")
+  @InBrowser(browser = Browser.FIREFOX, browserSize = "1920x1080")
+  public void mercury_search_navigateUsingSearchSuggestionsOnDesktop() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)